about.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/1bc3a7e061.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/about.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <!-- HEADER SECTION -->
+    <header class="header">
+      <div class="header-links">
+        <div class="header-left"></div>
+        <div class="header-right">
+          <ul class="header-link">
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-facebook"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-instagram"></i>
+              </a>
+            </li>
+          </ul>
+          <div class="header-items">
+            <a href="#">English</a>
+            <a href="#">Logout</a>
+          </div>
+        </div>
+      </div>
+      <!-- NAV SECTION -->
+      <nav>
+        <div class="nav-center">
+          <div class="nav-header">
+            <div class="logo">
+              <a href="index.html">
+                <img
+                  src="./assets/Images/conference-logo.png"
+                  alt="conference"
+                />
+              </a>
+            </div>
```

Comment:
As requested by the previous reviewer https://github.com/belaymit/Microverse-Capstone-Project-M1/pull/1#discussion_r1030811165, Please remove the extra whitespace on top :point_down: 
![image](https://user-images.githubusercontent.com/97521557/203716108-4c53ff86-f58f-4c12-9c13-3af7c4c3f9d6.png)